docs/dev/fundamentals/interacting.md

```diff
@@ -54,7 +54,11 @@ In order to add the zkSync Era alpha mainnet network to your wallet, you will ne
 
 ## Why are Metamask native contract interactions not working?
 
-It is not currently possible to interact with zkSync Era smart contracts via Metamask with EIP-1559 transactions. zkSync Era does not support EIP1559 transactions.
+It is not currently possible to interact with zkSync Era smart contracts via Metamask with EIP-1559 transactions. 
+
+zkSync Era supports the EIP-1559 transaction format but doesn't do anything with some of the transaction parameters.
```

Comment:
That is part of the answer to the question: https://matter-labs-workspace.slack.com/archives/C059ZDP49C5/p1685998989860449

Reply:
I don't know if this is the reason, but it sounds that I could interact with sc via mm and eip1559, just that the relevant fee fields will be ignored -- like the way it currently works with sending tokens for examples.

So either the question does not reflect reality to start with, or this part of the answer is not the reason why the interaction is impossible.

Reply:
Exactly that. But what is the question exactly? Why do we have 2 conflicting statements in the docs? And the answer is writer error, not enough info:

"zkSync Era does not support EIP1559 transactions." was incorrectly added, and it's not _quite_ true and needs expansion.

Reply:
If there's more to say, we should definitely say it,..

Reply:
https://linear.app/matterlabs/issue/DOC-409/why-exactly-does-mm-not-support-zksync-era-eip-1559-transaction

Reply:
Please remove the added sentence, or just don't address this ticket in this pr.

Reply:
why? what's wrong with it? It's helpful and then there's a link to the fuller explanation. We don't give the why in either case. This is more informative.

Reply:
It is misleading, and does not answer the question.
There's a question, then we say it is currently not possible, then we give a solution. I really don't see a reason to add more information.

> It's helpful and then there's a link to the fuller explanation

How is this helpful? It is not a fuller explanation for the question being asked. The ticket you opened indicates that we currently don't know the real explanation.

Reply:
it's better than what we had before

Reply:
better than what?
the incorrect statement should be removed, but nothing should be added.

Reply:
Metamask 'knows' how to work with eip1559 on our network.
You can try and do ETH transfer between 2 wallets, and Metamask will construct an EIP1559 tx for it.
You can actually also do a token transfer between 2 wallets, and the same will happen. Erc20-token transfer is in fact a contract interaction with the token contract.
In both cases, the priority fee is ignored. But, the tx will go through.

So if we don't support contract interaction via Metamask and eip1559, it is not because the fee fields are ignored. Therefore, I don't see reason to mention it.

I believe that this question is not relevant  anymore (for example, 1inch construct eip1559 tx). That is why I suggested to clarify it with the core team.

Reply:
OK, let me ask them and we can close this off ...

Reply:
This content is removed now. Sorry, I didn't have the context outside of the ticket.

Reply:
what's the reason?

Reply:
ask @githubdoramon  ...

Reply:
I don't know for sure, but i'm guessing it could even be a really badly written section which slipped through.. and something else entirely was intended ... who knows

Reply:
wait wait, do you know that now Metamask _can interact_ with smart contracts with eip1559 tx?

This question in the FAQ used to be valid. It goes back to the testnet, where indeed it was not possible to do so.